TopPlaces/TopPlaces/Model/PhotosHistory.h

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2016 Lightricks. All rights reserved.
+// Created by Gennadi Iosad.
+
+#import "PlacesPhotosProvider.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/// API for accessing persistent history storage.
+///
+@interface PhotosHistory : NSObject
```

Comment:
I see no reason why this has to be a static class. Static classes complicate tests, they introduce strong coupling between components and they break flexibility. These methods can be instance methods and you can achieve most of the benefits of static methods by making it a singleton either by exposing a static method that provide a shared instance (like <code>[UIApplication sharedApplication]</code>) or by providing the same instance via DI.